app_flutter/pubspec.yaml

```diff
@@ -34,6 +34,8 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  
```

Comment:
You can use the test methods exported from flutter_test, no need to add a package:test dependency.

Reply:
Switching to flutter_test caused the TypeMatcher test to fail.

```
    test('should return List<CommitStatus>', () {
      expect(service.getStats(), TypeMatcher<Future<List<CommitStatus>>>());
    });
```

Fails with:

```
  Expected: <Instance of 'TypeMatcher<Future<List<CommitStatus>>>'>
    Actual: <Instance of 'Future<List<CommitStatus>>'>
```

Do you have any idea why?

Reply:
Ahh, TypeMatcher was already a type in Flutter when it was added to package test, so that import is hidden in flutter_test. To use it you have to import package:test directly as you were doing - carry on!